mappings/net/minecraft/command/EntitySelector.mapping

```diff
@@ -30,14 +31,27 @@ CLASS net/minecraft/unmapped/C_nbqdyigm net/minecraft/command/EntitySelector
 	METHOD m_amntwqew getLimit ()I
 	METHOD m_argqjkpt usesAt ()Z
 	METHOD m_axqlnsgk getEntities (Lnet/minecraft/unmapped/C_vgpupfxx;Ljava/util/List;)Ljava/util/List;
+		ARG 1 vec
```

Comment:
Is this a position? If so, please change it to `pos`
```suggestion
		ARG 1 pos
```

Reply:
It is not, It is a Vec3

Reply:
Usually `Vec3`s are used for movement vectors or positions. `Vec3d` is used instead of `BlockPos` because the latter uses integers to represent a position, while the first uses doubles

Reply:
Ah so pos would be right then?